routes/shopifyApiProxy.js

```diff
@@ -18,7 +18,7 @@ module.exports = function shopifyApiProxy(request, response, next) {
 
   const fetchOptions = {
     method,
-    body,
+    body: request.read(),
```

Comment:
Do we need to do this anywhere else i.e. other routes?

Reply:
The only other place `body` appears in this package is here: https://github.com/Shopify/shopify-express/blob/master/middleware/webhooks.js#L5

Reply:
Should we make a simultaneous PR in `shopify-express` and have them referenced to each other? It's unfortunate that this will probably break existing apps consuming `shopify-express` but hey, it's beta 😬

Reply:
it's alpha :P

Reply:
have you tried this in an app? `request.read` doesn't work that way I don't think.

Reply:
Yup I tested and it created a product. Why do you think it only returns a single chunk and not the whole body?

Reply:
It's synchronous. If it does what you say that means we *really* don't want to use it because it will be tying up the thread.

Reply:
>The readable.read() method should only be called on Readable streams operating in paused mode. In flowing mode, readable.read() is called automatically until the internal buffer is fully drained.

> In general, it is recommended that developers avoid the use of the 'readable' event and the readable.read() method in favor of using either readable.pipe() or the 'data' event.

Reply:
Imo we should just add a middleware to routes that need the body that looks something like
```
app.use(function(req, res, next) {
  req.rawBody = '';
  req.setEncoding('utf8');

  req.on('data', function(chunk) { 
    req.rawBody += chunk;
  });

  req.on('end', function() {
    next();
  });
});
```